Files/MultilingualResources/Files.zh-Hans.xlf

```diff
@@ -1353,603 +1347,603 @@
+          <target state="translated">弹出设备时出现问题</target>
         </trans-unit>
         <trans-unit id="EjectNotificationErrorDialogBody" translate="yes" xml:space="preserve">
           <source>This device is currently in use. Close any programs, windows or tabs that might be using the device, and then try again.</source>
-          <target state="new">This device is currently in use. Close any programs, windows or tabs that might be using the device, and then try again.</target>
+          <target state="translated">该设备正在使用中。关闭可能使用该设备的所有程序，窗口或标签页，然后重试。</target>
         </trans-unit>
         <trans-unit id="SideBarEjectDevice.Text" translate="yes" xml:space="preserve">
           <source>Eject</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">弹出</target>
+          <target state="translated">弹出</target>
         </trans-unit>
         <trans-unit id="HorizontalMultitaskingControlDuplicateTab.Text" translate="yes" xml:space="preserve">
           <source>Duplicate tab</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">重复打开选项卡</target>
+          <target state="translated">复制选项卡</target>
         </trans-unit>
         <trans-unit id="HorizontalMultitaskingControlMoveTabToNewWindow.Text" translate="yes" xml:space="preserve">
           <source>Move tab to new window</source>
-          <target state="new">Move tab to new window</target>
+          <target state="translated">将标签页移动到新窗口</target>
         </trans-unit>
         <trans-unit id="HorizontalMultitaskingControlNewTab.Text" translate="yes" xml:space="preserve">
           <source>New tab</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">新选项卡</target>
+          <target state="translated">新建标签页</target>
         </trans-unit>
         <trans-unit id="NavSearchButton.AutomationProperties.Name" translate="yes" xml:space="preserve">
           <source>Search</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">搜索</target>
+          <target state="translated">搜索</target>
         </trans-unit>
         <trans-unit id="PropertiesDialogReadOnly.Text" translate="yes" xml:space="preserve">
           <source>Read-only</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">只读</target>
+          <target state="translated">只读</target>
         </trans-unit>
         <trans-unit id="PropertiesDialogHidden.Text" translate="yes" xml:space="preserve">
           <source>Hidden</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">隐藏</target>
+          <target state="translated">隐藏</target>
         </trans-unit>
         <trans-unit id="PropertyRatingText" translate="yes" xml:space="preserve">
           <source>Rating Text</source>
-          <target state="new">Rating Text</target>
+          <target state="translated">分级文本</target>
```

Comment:
What does these `Text` mean here?

Reply:
@winston-de Can you help with this?

Reply:
@linsui It is meant to tell the developer that the value is displayed to the user, and should not be parsed programmatically. It says that because the names were generated from its property system name, which is "System.RatingText". It can probably be ignored/removed.

Reply:
Thanks! I fixed the translation. Should I fix the source?

Reply:
@linsui If you want to you can. I'll be opening a PR to fix a few of the other names, too, so you don't have to.

Reply:
Ok. I'll keep it as it is.